### DISCUSSION TITLE
aria-expanded=false

### DISCUSSION THREAD

**Discussion by tarik916** [tabpanelwidget/tabpanelwidget#13] [Ideas] [answered]
Hi. Thank you for writing this great plugin! 

When testing with a screen reader, it identifies the open accordions but cannot identify the closed ones as collapsed buttons since they do not have an expanded status unless opened. I added my own js to add the closed status but the screen reader does not announce closed if the user hits enter, only if the user uses a mouse. Can this be updated for better accessibility?

**Comment by thierryk** [comment, marked as answer]
Thank you for reporting this.

The state of the  `aria-expanded` attribute should be **explicit** (`true` / `false`).
Per spec, when it is **undefined (default)**:

> The element, or another grouping element it controls, is neither expandable nor collapsible; all its child elements are shown or there are no child elements.

We'll update the script.

**Reply by tarik916** [reply]
Yes, I should have said "false" not closed. Thank you for considering the update.

**Comment by thierryk** [comment]
Thanks @tarik916, the fix is in [1.4.0](https://github.com/tabpanelwidget/tabpanelwidget/releases/tag/1.4.0)
